tests/test_correlation_functions.py

```diff
@@ -81,7 +82,35 @@ def test_getCF(self):
         results3 = realshear(lens_data, source_data, config=stile_args)
         numpy.testing.assert_equal(results, results3)
         
-        
+    def test_plot(self):
```

Comment:
The `test_plot` works for me, but `test_getCF` is broken unfortunately at the moment